### PR TITLE
Update editors list and acknowledgements

### PR DESCRIPTION
**Pull request by sebastiankb**
In today's TD call I like to talk about editors list for TD 1.1 + acknowledgements.

Please let me know if I missed someone.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1718.html" title="Last updated on Oct 12, 2022, 4:08 PM UTC (f2f2420)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1718/5c1822e...f2f2420.html" title="Last updated on Oct 12, 2022, 4:08 PM UTC (f2f2420)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,25 +32,34 @@
...
                 {
                     name:       "Michael McCool",
                     w3cid:      "93137",
                     company:    "Intel",
                     companyURL: "https://www.intel.com/"
                 },
                 {
-                    name:       "Victor Charpenay",
-                    w3cid:      "84448",
+                    name:       "Ege Korkan",
```
Comment (egekorkan) [inline index.html:42]: ah thank you!

w3cid: "110131" is my w3c identification

File: index.html [hunk 2]
```diff
@@ -8265,7 +8275,7 @@ <h2 id="changes-from-fpwd">Changes from First Public Working Draft 24 November 2
 
 <section id="acknowledgements" class="appendix normative">
   <h1>Acknowledgements</h1>
-  <p>The editors would like to special thank Matthias Kovatsch (co-editor of TD 1.0), Michael Koster, Michael Lagally, Kazuyuki Ashimura, Ege Korkan, Daniel Peintner, Toru Kawaguchi, María Poveda, Dave Raggett, Kunihiko Toumura, Takeshi Yamada, Ben Francis, Manu Sporny, Klaus Hartke, Addison Phillips, Jose M. Cantera, Tomoaki Mizushima, Soumya Kanti Datta and Benjamin Klotz for providing contributions, guidance and expertise.</p>
+  <p>The editors would like to special thank Thomas Jäckle, Jan Romann, Elodie Thiéblin, Michael Koster, Michael Lagally, Kazuyuki Ashimura, Daniel Peintner, Toru Kawaguchi, María Poveda, Dave Raggett, Kunihiko Toumura, Takeshi Yamada, Ben Francis, Manu Sporny, Klaus Hartke, Addison Phillips, Jose M. Cantera, Tomoaki Mizushima, Soumya Kanti Datta and Benjamin Klotz for providing contributions, guidance and expertise.</p>
```
Comment (danielpeintner) [inline index.html:8278]: Matthias Kovatsch removed by accident?

Comment (sebastiankb) [reply index.html:8278]: Matthias moved to the former editor list.

**Comment by sebastiankb** [comment]
from today's TD call: group decided to merge, however we need to check with @mkovatsc if he wants to show up as Huawei or as Siemens employee. If this is clarified @sebastiankb will merge this PR.

**Comment by sebastiankb** [comment]
I just spoke with @mkovatsc and he suggest to keep Huawei affiliation to keep consistency with editor list information of the TD 1.0. 

I'm going to merge this PR.

**Pull request merged**